libtracepoint-decode-cpp/include/tracepoint/PerfEventInfo.h

```diff
@@ -38,6 +39,7 @@ namespace tracepoint_decode
     {
         PerfEventDesc const* event_desc;        // Always valid if GetSampleEventInfo() succeeded.
         PerfEventSessionInfo const* session_info;//Always valid if GetSampleEventInfo() succeeded.
+        perf_event_header const* header;        // Always valid if GetSampleEventInfo() succeeded. Points into event.
```

Comment:
Note: This is the data that we provide to callers when they enumerate a trace file or when they enumerate the events in a trace buffer. I've added a pointer to the raw event data -- usually not needed, but occasionally useful. For example, if you want to have a real-time session and you want to stream the events into a file, you would need this.